rewrite-core/src/main/java/org/openrewrite/GitRemote.java

```diff
@@ -279,7 +279,7 @@ private String repositoryPath(RemoteServerMatch match, URI normalizedUri) {
                     .replaceFirst("^/", "");
         }
 
-        private static final Pattern PORT_PATTERN = Pattern.compile(":\\d+");
+        private static final Pattern PORT_PATTERN = Pattern.compile(":\\d+[^-\\w]"); // Allow orgs with numbers in name
```

Comment:
github at least supports org names with only numbers.

Reply:
Guess those will still get the exception thrown then, but at least this works for partial numeric org names. Welcome suggestions for a complete fix!

Reply:
> Guess those will still get the exception thrown then, but at least this works for partial numeric org names. Welcome suggestions for a complete fix!

I had opened this previously https://github.com/openrewrite/rewrite/pull/4651

Reply:
Ah, I had no idea. Approved just now & sorry about the conflict. Do let me know if there's ever something open for too long.